### ISSUE TITLE
sev 3- all - Unnecessary screenreader recitation on Accordion Expanded focus

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9445]
## What happened?

When reviewing the upcoming changes to claims with the BVA / Section 508, we discovered some confusing / misleading recitations that we want to clean up. 

## Specs:

- Device: All
- OS:
- User Account:
- Accessibility State:

## Steps to Reproduce

Using a screenreader, navigate to the Claim Details screen for any claim. Tab to an expanded accordion. Here, there is unnecessary information being read out by the screenreader ("review more details").
## Desired behavior

Accessibility recommendation is to remove the "review more details" SR text on Accordion Expanded. 

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low

## Linked to Story

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by alexandec** [comment]
Expanding the scope of this ticket to cover all uses of AccordionCollapsible, as follows:

- BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimTimeline/ClaimPhase.tsx
- HealthScreen/Appointments/NoMatchInRecords/NoMatchInRecords.tsx
- HealthScreen/SecureMessaging/ViewMessage/CollapsibleMessage.tsx

**Comment by alexandec** [comment]
See [slack discussion](https://adhoc.slack.com/archives/C067S0FRP0C/p1724343995185379) for additional details. Will tackle this along with [#9346](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/9346).

**Comment by brea11y** [comment]
@alexandec – Is this ready for review?

**Comment by alexandec** [comment]
@brea11y yep! And #9346 (the collapse version) is also. They're both covered by the same PR, #9450.

**Comment by brea11y** [comment]
Perfect! Thank you! I'll take a look right now.

**Comment by alexandec** [comment]
@rbontrager same question on this ticket, any additional QA needed?

**Comment by TKDickson** [comment]
No further testing needed in develop; closing
